t/51-schema.t

```diff
@@ -50,5 +50,11 @@ dies-ok {
     $s1.Person.^create: :name<bla>;
 }
 
+my $s3 = schema( Z => 'Post', X => 'Person');
```

Comment:
Would you mind to just pass on of those models as Type object instead of string? just to test if that's working?

Reply:
Sure.

Reply:
There.

Reply:
Thank you!